flink-table/flink-sql-parser/pom.xml

```diff
@@ -162,11 +163,6 @@ under the License.
 					<groupId>com.google.uzaygezen</groupId>
 					<artifactId>uzaygezen-core</artifactId>
 				</exclusion>
-				<!-- Excluding org.apiguardian:apiguardian-api v1.1.0 because org.junit.jupiter:junit-jupiter contains newer version v1.1.2 -->
-				<exclusion>
-					<groupId>org.apiguardian</groupId>
-					<artifactId>apiguardian-api</artifactId>
-				</exclusion>
```

Comment:
Calcite 1.31.0 now depends on 1.1.2 api.guardian (same as junit-jupiter) and since the latest commit to api.guardian was about 2 years ago[1] it's pretty unlikely there will be new versions...

[1] https://github.com/apiguardian-team/apiguardian